legal-api/src/legal_api/services/filings/validations/agm_extension.py

```diff
@@ -39,6 +41,15 @@ def validate(business: Business, filing: Dict) -> Optional[Error]:
     if business.legal_type not in enabled_filings:
         return Error(HTTPStatus.FORBIDDEN,
                      [{'error': babel(f'{business.legal_type} does not support agm extension filing.')}])
+    authorized_permissions = PermissionService.get_authorized_permissions_for_user()
+    if validate_certify_name(filing):
+        allowed_role_comments = ListActionsPermissionsAllowed.EDITABLE_CERTIFY_NAME.value
```

Comment:
When a user has this permission, they can write whatever they want in the Certify Name field.

When they don't have this permission, the field is pre-populated and uneditable.

I don't a check here for whether the user edited the Certify Name field.